deploy/cron_scripts/update_latest.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+# This script runs the machinery in the given dir to scrape latest.json
+# using the Hydrant scrapers, then updates it into the given directory.
+
+# This script is called with two arguments: the Hydrant base dir, and the
+# output dir.
+[ ! -z "$2" ] || {
+	echo "Usage: $0 <Hydrant base dir> <output dir>"
+	exit 11
+}
+
+# Hydrant base directory, i.e. the one that has the copy of the repo, with
+# all the scripts in it. In the locker, this is ~/web_scripts/updating/hydrant
+DIR="$1"
+
+# The output directory, i.e. the one that has the folder being served to the
+# internet. In the locker, this is ~/web-scripts/hydrant
+OUT="$2"
+
+cd "$DIR/scrapers"
+
+# -q means quietly; don't report anything in stdout or stderr.
+git pull -q
```

Comment:
Do we want to add a guard to this, like
```
git checkout -f deploy
```
? I'm worried in particular that this will break if someone shells in and modifies files locally ("Aborting merge; you'll overwrite local changes")

Reply:
that makes sense, i'll add this